app/setting/workspaceprofile/page.tsx

```diff
@@ -20,8 +20,23 @@ import {
 import Button from '@/components/button/button';
 import Input from '@/components/input/input';
 import SVGIcon from '@/assets/icons/SVGIcon';
+import { useStores } from '@/stores';
+import { updateWorkspaceDetails } from '@/services/clientSide/workspaceServices';
 
 function WorkspaceProfile() {
+  const { workspaceStore } = useStores();
+  const { currentWorkspace } = workspaceStore;
+  const [organizationName, setOrganizationName] = useState(
+    currentWorkspace?.name,
+  );
+
+  const handleUpdate = (e: SyntheticEvent) => {
+    e.preventDefault();
+    if (currentWorkspace)
+      workspaceStore.setCurrentWorkspace({ ...currentWorkspace });
+    if (organizationName) updateWorkspaceDetails(organizationName);
```

Comment:
if (organizationName) updateWorkspaceDetails(organizationName);

when we call service for any form then we will create const for payload and pass that
const payload = { name: organizationName }
updateWorkspaceDetails(payload);

Reply:
Done